src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.html

```diff
@@ -5,10 +5,17 @@
 <cd-info-panel *ngIf="!grafanaExist"
                i18n>Please consult the
   <a href="{{ docsUrl }}" target="_blank">documentation</a> on how to
-  configure and enable the monitoring functionality.</cd-info-panel>
+  configure and enable the monitoring functionality.
+</cd-info-panel>
+
+<cd-info-panel *ngIf="!dashboardExist"
+               i18n>Dashboard does exist. Please refer to
+  <a href="{{ docsUrl }}" target="_blank">documentation</a> on how to
+  add dashboards to Grafana.
```

Comment:
Please close the element in the same line as the string.
```suggestion
  add dashboards to Grafana.</cd-info-panel>
```